samples/test/solution-counters.test.js

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const { execSync } = require('child_process');
+const { assert } = require('chai');
+const exec = cmd => execSync(cmd, { encoding: 'utf8' });
+
+describe('distributed counter', () => {
+    it('should create, increase and get counter', () => {
+        const output = exec('node solution-counters.js');
```

Comment:
Hm. Can you rewrite this test to call `incrementCounter()` and `getCount()` directly? That lets you verify the output without string comparisons.

Reply:
For nodejs the samples are designed like user can run the file independently without any code changes and can see the output.
all the function will calls the internally, and in test file we are just executing the sample file and matching the output from the sample, the is same in all the repository.